tests/test_csc.py

```diff
@@ -1058,10 +1079,11 @@ async def test_switchForceBalanceSystem(self) -> None:
                 flush=True, timeout=STD_TIMEOUT
             )
 
-            # LUT gravity and temperature absolute values should be larger
-            # than zero
+            # LUT gravity absolute value should be larger than zero
             self.assertTrue(np.all(np.abs(axial_forces.lutGravity) > 0.0))
-            self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) > 0.0))
```

Comment:
You might want to add max value as well. The temperature offsets shall be a few N, so something like:
```
self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) > 0.0))
self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) < 20.0))
```

or even:
```
self.assertTrue(np.all(np.abs(10 > axial_forces.lutTemperature) > 0.0))
```

Reply:
The line change of `1064` was removed actually. Thanks!